src/definitions/modules/calendar.js

```diff
@@ -211,6 +212,20 @@ $.fn.calendar = function(parameters) {
           }
         },
 
+        trigger: {
+          change: function() {
+            var
+                events       = document.createEvent('HTMLEvents'),
```

Comment:
I think `events` should be created inside `if`, so it wouldn't consume CPU and memory when `inputElement` not found.

Reply:
I agree, but for consistency i will change this in checkbox/dropdown as well (because the method is the same there)

Reply:
Done by 4d6363fde4198b67f9c779c2bd7d4913846247f9 🙂